src/WebServer.Logging/WebServer.Logging.csproj

```diff
@@ -0,0 +1,134 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Because I started doing an implementation and I think almost everyone will write a class like that, it's a helpful thing to be used by the one implementing a specific Logger, you can simply implement two methods to make it work.